mesonbuild/modules/external_project.py

```diff
@@ -93,10 +93,10 @@ def __init__(self,
         # will install files into "c:/bar/c:/foo" which is an invalid path.
         # Work around that issue by removing the drive from prefix.
         if self.prefix.drive:
-            self.prefix = self.prefix.relative_to(self.prefix.drive)
+            self.prefix = Path(relpath(str(self.prefix), str(self.prefix.drive)))
```

Comment:
relpath accepts pathlib objects, no need to cast to str() here.

Reply:
Done at b12e1df